LaunchUpCorp/Focus-Finder
# feat(FOC-17): set up frontend react website

## Pull Request Title (Using conventional commit Standard)

### Type of Change


### Description

- Set up frontend dashboard website to show the users statistics of their activities. 
  - Tech stack
    - Vite
    - React
    - TypeScript
    - TailwindCSS
    - GraphQL with Apollo server
- GraphQL API is set up and a test query to the backend is made on the main page.
```
  query {
    user(id: 1) {
      id
      email
    }
  }
```
<img width="339" alt="Screenshot 2023-05-03 at 10 57 50 PM" src="https://user-images.githubusercontent.com/94641590/236103091-1de26d88-2a95-4e1d-abbf-da6f86064c48.png">


### Testing Instructions

1. Run `npm install` to install dependencies.
2. To test GraphQL queries, create `.env` file and add the values in `.env.example`.
3. Make sure the backend server is running to make GraphQL queries.
4. Run `npm start` to start the project.

### Checklist (delete irrelevant items)

- [x] I have followed coding standards and best practices.
- [x] I have updated the relevant tests if needed.
